x-pack/test/functional/apps/lens/group6/metric.ts

```diff
@@ -136,7 +128,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
           extraText: 'Average of bytes 19,755',
           value: '19,755',
           color: 'rgba(245, 247, 250, 1)',
-          showingTrendline: false,
+          showingTrendline: true,
```

Comment:
modified this test to test the bug Drew mentioned.

Reply:
Great idea 👍